fgrpc/grpcrunner_test.go

```diff
@@ -30,58 +30,271 @@ import (
 	"google.golang.org/grpc/health/grpc_health_v1"
 )
 
+var (
+	// Generated from "make cert"
+	caCrt  = "../ca.crt"
```

Comment:
we should probably use the mostly standard testdata/ for location of the certs (ie `fortio/fgrpc/testdata/ca.crt` etc...) but that's fine too I guess

Reply:
ack